flixel/FlxCamera.hx

```diff
@@ -786,6 +795,8 @@ class FlxCamera extends FlxBasic
 		updateFade(elapsed);
 		updateShake(elapsed);
 		
+		flashSprite.filters = enableFilters ? _filters : null;
```

Comment:
Is this safe performance-wise? I'd imagine this would call `set_filters()` each frame.